### DISCUSSION TITLE
Where'd the export to plantUML plugin go?

### DISCUSSION THREAD

**Discussion by robert-STA** [jgraph/drawio#4616] [Q&A] [answered]
Hi, there use to be a drawio plugin to export drawio diagrams to plantUML. It was previously located "https://jgraph.github.io/drawio-plantuml/plantuml.js" but that no longer exists. Does anyone have a copy of the plugin or know where I can get it? Thanks!

**Comment by davidjgraph** [comment, marked as answer]
We've never had any functionality that converted from draw.io format to PlantUML format.
